default.yml

```diff
@@ -20,6 +25,16 @@ AllCops:
 Rails:
   Enabled: true
 
+Codeur/RailsAppPatterns:
+  Enabled: true
+  ForbiddenPatterns:
+    - presenters
+    - view_objects
+    - modules
```

Comment:
Do we really want to forbid modules ? Where would you move it instead ?

Reply:
Most of them seems to be created as concerns (before concerns exist in Rails).
To be clear, I talk about `app/modules` in Codeur, and not the modules in general.

Reply:
Ok perfect